cmd/gpu_nfdhook/labeler_test.go

```diff
@@ -264,12 +268,7 @@ func TestLabeling(t *testing.T) {
 			if tc.expectedRetval == nil && !reflect.DeepEqual(labeler.labels, tc.expectedLabels) {
 				t.Errorf("test %v label mismatch with expectation:\n%v\n%v\n", tc.name, labeler.labels, tc.expectedLabels)
 			}
-			for filename := range tc.capabilityFile {
-				os.Remove(path.Join(root, filename))
-			}
-			for filename := range tc.sysfsfiles {
-				os.Remove(path.Join(sysfs, filename))
-			}
+			os.RemoveAll(subroot)
```

Comment:
this leaves `subroot` if the case fails earlier? maybe `t.Cleanup(func() { os.RemoveAll(subroot) })` after `createFiles()`?

Reply:
No need to. The root cleanup will clean those up.

Reply:
> The root cleanup will clean those up.

indeed, +1 for dropping L271

Reply:
dropped